### PR TITLE
Fix logo SVG

### LINKED ISSUE #253: Favicon
Let's create a favicon for so that the site is recognizable in tabs and bookmarks. Let's use the logo for this icon.

### LINKED ISSUE #283: Font in logo is wrong
On some platforms (Linux and probably Windows), Lucida Grande is not available and our logo does not display correctly.

screenshot from 2018-08-02 14-53-45

Let's load the logo on a platform that does have Lucida Grande, convert the text to a path, and re-save the logo.

### PR DESCRIPTION
**Pull request by rdebeasi**
**What issue does this PR solve?**
Resolves #283
Resolves #253

**Explain the problem and the proposed solution**
- Typeset in Arino, not Lucida Grande
- Converted object to path so that font isn't required to view SVG
- Made it bigger on med/large screens
- Removed old unused images
- Fixed minor vertical alignment weirdness in logo
- Generate favicon and touch icon files from logo

[Updated logo](https://github.com/openpracticelibrary/openpracticelibrary/blob/5dd5aedbe967b4bf027ee59603f0a3f6652bd9de/static/images/logo.svg)

@Rileyi, let me know if you have any thoughts on these minor updates to the logo. Thank you!

### CODE REVIEW

File: static/css/custom.css [hunk 1]
```diff
@@ -370,6 +370,10 @@ a:focus {
   .tile-heading {
     font-size: 2.2rem;
   }
+
+  .blog-logo img {
+    height: 50px;
```
Comment (springdo) [inline static/css/custom.css]: @rdebeasi  - by setting this height here on mobile view the height remains and looks a bit big compared to the menu bar across from it. I am OK with this, but other people might not be? just a thought anyway.

**Comment by rdebeasi** [comment]
Thanks for catching that! I can fix and am happy to defer merging until that bug is fixed.

**Comment by rdebeasi** [comment]
As we noticed on our call, it's a pre-existing Chrome-specific bug. Weird!

Anyway, it's fixed now!

**Comment by springdo** [comment]
Thanks @rdebeasi, I'll mergé

**Pull request merged**